fixtures/artifactor_plugin.py

```diff
@@ -58,7 +58,7 @@ def get_test_idents(item):
 
 def get_name(obj):
     if hasattr(obj, '_param_name'):
-        return getattr(obj, obj._param_name)
+        return obj._param_name
```

Comment:
can we condense this to `getattr(obj,  '_param_name', None) or getattr(obj, 'name', None) or str(obj)`